### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by jemutakiplagat**
-Added @media queries for responsiveness
-modified some css to fit wide screen

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -11,16 +11,19 @@ main {
...
+/* .full-wrapper {
   position: absolute;
   width: 375px;
   height: 718px;
   left: 1px;
   background-color: #fffffffa;
   align-items: center;
-}
+  background-color: #396DF2;
+} */
```
Comment (CollinsTatang) [inline index.css:26]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.

File: index.css [hunk 2]
```diff
@@ -906,3 +619,42 @@ textarea[id=subject] {
...
+  .contact-container {
+    flex-direction: row;
+    width: 1000px;
+  }
+
+  /* .info{
+    flex-direction: row;
+    width: 1000px;
+    display: flex;
+  } */
```
Comment (CollinsTatang) [inline index.css:634]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.

File: index.css [hunk 3]
```diff
@@ -321,318 +234,153 @@ a {
...
-.work-section2 > .frame {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 0;
-  position: static;
-  width: 285px;
-  height: 32px;
+  /* width: 195px;
+  height: 72px; */
```
Comment (CollinsTatang) [inline index.css:317]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.

File: index.html [hunk 4]
```diff
@@ -34,7 +33,7 @@
             </nav>
             
         </header>
-        </div>
+
```
Comment (CollinsTatang) [inline index.html:36]: - [ ] Please work on your desktop design view following the Figma [Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
### **Take a look at your design**
head
### **This is Figmal Design**
destop

File: index.css [hunk 5]
```diff
@@ -11,16 +11,19 @@ main {
   width: 375px;
   height: 5410px;
   background: #fff;
+  margin-left: 30px;
+  display: grid;
```
Comment (CollinsTatang) [inline index.css:15]: - [ ] Please work on your mobile design to still look as expected. The project is no more responsive at any screen size
### **Take A Look at Your Mobile Design**
mobile

File: index.css [hunk 6]
```diff
@@ -161,7 +166,8 @@ a {
   margin-left: -10px;
 }
 
-.work-section {
+.card {
+  box-sizing: border-box;
   display: flex;
```
Comment (CollinsTatang) [inline index.css:171]: - [ ] Please used the media query for the Desktop breakpoint in your CSS.
res

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @jemutakiplagat, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 


- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jemutakiplagat** [comment]
-Made the page responsive 
-Removed all comments to make my code clean

**Comment by arslanbisharat** [comment]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**